src/server.rs

```diff
@@ -144,7 +180,17 @@ impl Server
 
     fn router_webui(&self) -> nickel::Router {
         let mut router = Nickel::router();
-        router.get("/", handler_vuejs);
+        router.get("/", middleware_webui);
+        router.get("/api/ui/version", middleware!(format!("Lucid Version {}", crate_version!())));
+        router
+    }
+
+    fn router_sse(&self) -> nickel::Router {
+        let mut router = Nickel::router();
+        router.get("/sse/test", middleware! { |_request, mut response|
+            response.set(StatusCode::BadRequest).set(MediaType::Json);
+            "lol"
```

Comment:
lol

Reply:
Trying to implement SSE server, but it will be squashed.